bot/exts/utils/attachment_pastebin_uploader.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import re
+
+import aiohttp
+import discord
+from discord.ext import commands
+from pydis_core.utils import paste_service
+
+from bot.bot import Bot
+from bot.constants import Emojis
+from bot.log import get_logger
+
+log = get_logger(__name__)
+
+PASTEBIN_UPLOAD_EMOJI = Emojis.check_mark
+DELETE_PASTE_EMOJI = Emojis.trashcan
+
+
+class EmbedFileHandler(commands.Cog):
+    """
+    Handles automatic uploading of attachments to the paste bin.
+
+    Whenever a user uploads one or more attachments that is text-based (py, txt, csv, etc.), this cog offers to upload
+    all the attachments to the paste bin automatically. The steps are as follows:
```

Comment:
Alignment of bullet items in this documentation list is wrong.

Reply:
Where would I find an example of them being used correctly?